Makefile

```diff
@@ -0,0 +1,204 @@
+ifneq (,)
+.error This Makefile requires GNU Make.
+endif
+
+.PHONY: help gen lint test _gen-main _gen-examples _gen-modules _lint_files _lint_fmt _pull-tf _pull-tf-docs
+
+CURRENT_DIR     = $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+TF_EXAMPLES     = $(sort $(dir $(wildcard $(CURRENT_DIR)examples/*/)))
+TF_MODULES      = $(sort $(dir $(wildcard $(CURRENT_DIR)modules/*/)))
+
+TF_VERSION      = light
```

Comment:
Could we pin the version to `0.12.16`?

Reply:
From my POV. It's fine like this, if it fails it fails and that's a good moment to see what needs to be updated.